Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -46,6 +76,27 @@ protected function getAttributeValue($object, $attribute, $format = null, array
         return $this->attributeExtractionStrategy->getValue($object, $attribute, $format, $context);
     }
 
+    /**
+     * @param object|string $classOrObject
+     * @param string        $attribute
+     * @param null          $format
+     * @param array         $context
+     *
+     * @return bool
+     */
+    protected function isAllowedAttribute($classOrObject, $attribute, $format = null, array $context = array())
```

Comment:
Or

    protected function isAllowedAttribute($classOrObject, $attribute, $format = null, array $context = array())
    {
        $isAllowed = parent::isAllowedAttribute($classOrObject, $attribute, $format, $context);

        $access = $this->reader->getPropertyMetadata($classOrObject, $attribute, Access::class);

        if ($access instanceof Access && ($attributes = $access->value)) {
            $attributes = is_array($attributes) ? $attributes : [$attributes];

            return $isAllowed && $this->authorizationChecker->isGranted($attributes, $context['object']);
        }

        return $isAllowed;
    }